docs/source/control_tutorials.rst

```diff
@@ -195,6 +195,25 @@ constrained to the allocated region. ::
     >>> block.read(13)
     b"Hello, world!"
 
+.. warning::
+    Writes are buffered before being transmitted to the SpiNNaker board.
+    :py:meth:`~.rig.machine_control.machine_controller.MemoryIO.flush` must be
+    called to force writes on a given file-like (and its siblings -- see below)
+    to be completed.
```

Comment:
Is there no way to enable unbuffered mode? (Also, shouldn't that be default -- enabling buffering is something to do only when you know what you're doing and care about performance, I'd hope).